CMakeLists.txt

```diff
@@ -155,6 +159,10 @@ foreach(OPTIONS_MODULE ${OPTIONS_MODULES})
   endif()
 endforeach()
 
+if(CA_NATIVE_CPU)
+  set(CA_RUNTIME_COMPILER_ENABLED Off)
```

Comment:
it would be good to have a comment explaining why you needed to do this (it's slightly counter-intuitive, but ultimately I think it relates to what needs to be built).

Reply:
I've added the comment, thank you